e2e/js_image_docker/WORKSPACE

```diff
@@ -31,46 +31,10 @@ load("@npm//:repositories.bzl", "npm_repositories")
 
 npm_repositories()
 
-###
-# Setup rules_pkg
-###
-http_archive(
-    name = "rules_python",
-    sha256 = "a868059c8c6dd6ad45a205cca04084c652cfe1852e6df2d5aca036f6e5438380",
-    strip_prefix = "rules_python-0.14.0",
-    url = "https://github.com/bazelbuild/rules_python/archive/refs/tags/0.14.0.tar.gz",
-)
-
-http_archive(
-    name = "rules_pkg",
-    sha256 = "eea0f59c28a9241156a47d7a8e32db9122f3d50b505fae0f33de6ce4d9b61834",
-    urls = ["https://github.com/bazelbuild/rules_pkg/releases/download/0.8.0/rules_pkg-0.8.0.tar.gz"],
-)
-
-load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
-
-rules_pkg_dependencies()
-
 ###
 # Setup rules_docker
 # rules_docker does not work with M1 macs due to ancient rules_go version.
 ###
-
-# WORKAROUND START
-# See: https://github.com/bazelbuild/rules_docker/pull/2035
-http_archive(
-    name = "io_bazel_rules_go",
-    sha256 = "099a9fb96a376ccbbb7d291ed4ecbdfd42f6bc822ab77ae6f1b5cb9e914e94fa",
-    urls = ["https://github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip"],
-)
-
-load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
-
-go_rules_dependencies()
-
-go_register_toolchains(version = "1.19.3")
-# WORKAROUND END
-
```

Comment:
Not sure if it was intentional to remove this workaround, but I found the build wouldn't work on my M1 mac without it. Just wanted to make sure you knew; let me know if there's a better workaround that I'm missing